app/src/main/java/me/saket/dank/ui/subreddit/SubredditActivity.java

```diff
@@ -425,6 +432,14 @@ private void setupToolbarSheet() {
     });
   }
 
+  private void loadLastSortingAndPeriod(String subredditName) {
+    SortingAndTimePeriod sort = userSessionRepository.get().getSortingAndTimePeriodForSub(subredditName);
+    if (sort == null)
```

Comment:
Braces with if and else please? 

I'll soon add some static analyzer to the project so that code formatting comments do not leak into code reviews.